std/evmprecompiles/bn_test.go

```diff
@@ -127,10 +123,7 @@ func TestECMulCircuitFull(t *testing.T) {
 	t.Skip("skipping very long test")
 	assert := test.NewAssert(t)
 	circuit, witness := testRoutineECMul(t)
-	assert.ProverSucceeded(circuit, witness,
-		test.NoFuzzing(), test.NoSerialization(),
-		test.WithBackends(backend.GROTH16, backend.PLONK), test.WithCurves(ecc.BN254),
-	)
+	assert.CheckCircuit(circuit, test.WithValidAssignment(witness))
```

Comment:
I think in addition to changing from `ProverSucceeded` to `CheckCircuit`, we can also enable the test. Previously as there wasn't a way to distinguish between release test and PR test, I had it skipped. But now I think it would be OK to re-enable.